plugins/source/aws/resources/services/iam/saml_identity_providers.go

```diff
@@ -32,3 +36,31 @@ func SamlIdentityProviders() *schema.Table {
 		},
 	}
 }
+
+func fetchIamSamlIdentityProviders(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	svc := meta.(*client.Client).Services().Iam
+	response, err := svc.ListSAMLProviders(ctx, &iam.ListSAMLProvidersInput{})
+	if err != nil {
+		return err
+	}
+
+	res <- response.SAMLProviderList
+	return nil
+}
+
+func getSamlIdentityProvider(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource) error {
+	svc := meta.(*client.Client).Services().Iam
+	p := resource.Item.(types.SAMLProviderListEntry)
+
+	providerResponse, err := svc.GetSAMLProvider(ctx, &iam.GetSAMLProviderInput{SAMLProviderArn: p.Arn})
+	if err != nil {
+		return err
+	}
+
+	resource.Item = models.IAMSAMLIdentityProviderWrapper{
+		GetSAMLProviderOutput: providerResponse,
+		Arn:                   *p.Arn,
```

Comment:
no real opinion, but maybe `aws.StringValue`?

Reply:
I am not trying to fix or change the code, just trying to change the structure/formatting the files